src/refer.ts

```diff
@@ -64,6 +90,52 @@ function refer(this: any, options: any) {
     }
   }
 
+  async function actLostEntry(this: any, msg: any) {
+    const seneca = this
+
+    const entryList = await seneca.entity('refer/occur').list$({
+      email: msg.email,
+      kind: 'create',
+    })
+    entryList.forEach((entry: any) => {
```

Comment:
in general, place seneca operations inside for loops, so you can use await inside them to ensure all operations complete before returning

Reply:
Indeed, thanks for let me know. Done!